src/main/java/nextstep/courses/domain/session/Enrollment.java

```diff
@@ -1,36 +1,41 @@
 package nextstep.courses.domain.session;
 
+
+import java.util.OptionalLong;
+
 import nextstep.courses.CannotEnrollException;
 import nextstep.courses.domain.Amount;
 import nextstep.payments.domain.Payment;
 
+/**
+ * 수강 신청 클래스
+ */
+
 public class Enrollment {
-    private SessionStatus status;
-    private final EnrollmentPolicy enrollmentPolicy;
     private int enrolledCount = 0;
+    private final EnrollmentPolicy enrollmentPolicy;
 
-    public Enrollment(SessionStatus status, EnrollmentPolicy enrollmentPolicy) {
-        this.status = status;
+    public Enrollment(EnrollmentPolicy enrollmentPolicy) {
         this.enrollmentPolicy = enrollmentPolicy;
```

Comment:
```suggestion
        this(enrollmentPolicy, 0);
```
부생성자가 주생성자를 호출하도록 구현하는 것은 어떨까?